application/src/main/java/org/opentripplanner/transit/api/request/RegularStopRequest.java

```diff
@@ -0,0 +1,40 @@
+package org.opentripplanner.transit.api.request;
+
+import javax.annotation.Nullable;
+import org.locationtech.jts.geom.Envelope;
+import org.opentripplanner.transit.model.site.RegularStop;
+
+/**
+ * A request for {@link RegularStop}s.
+ * <p/>
+ * This request is used to retrieve {@link RegularStop}s that match the provided criteria.
+ */
+public class RegularStopRequest {
+
+  private final Envelope envelope;
+  private final String agency;
```

Comment:
Maybe I don't get it, but shouldn't this be a `FeedScopedId`?

Reply:
Should be `String feedId`